plugins/github-actions/dev/index.tsx

```diff
@@ -14,7 +14,65 @@
  * limitations under the License.
  */
 
+import React from 'react';
 import { createDevApp } from '@backstage/dev-utils';
-import { githubActionsPlugin } from '../src/plugin';
+import { EntityGithubActionsContent, githubActionsPlugin } from '../src/plugin';
+import { EntityProvider } from '@backstage/plugin-catalog-react';
+import { Entity } from '@backstage/catalog-model';
+import { GithubActionsApi, githubActionsApiRef } from '../src';
+import getWorkflowRunResponse from './getWorkflowRun.json';
+import listJobsForWorkflowRunResponse from './listJobsForWorkflowRun.json';
 
-createDevApp().registerPlugin(githubActionsPlugin).render();
+const mockEntity: Entity = {
```

Comment:
Is there a way to import [sample.yaml](https://github.com/backstage/backstage/blob/master/plugins/github-actions/examples/sample.yaml) to get this entity instead of redefining it here?

Reply:
I tried importing the yaml file but got the error below. Rather than breaking the example directory pattern we have going, I'll just keep this mock defined inline

```
Error: You attempted to import ../examples/sample.yaml which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/
```

Reply:
Hm I'm not sure honestly, I guess it makes sense to not allow imports outside of the source dir. Could be worth duplicating the file inside the fixtures folder for this purpose.